include/aspect/geometry_model/box.h

```diff
@@ -30,17 +30,39 @@ namespace aspect
 {
   namespace GeometryModel
   {
+
     using namespace dealii;
 
     /**
      * A class that describes a box geometry of certain width, height, and
-     * depth (in 3d).
+     * depth (in 3d), and, possibly, topography.
      */
     template <int dim>
     class Box : public Interface<dim>, public SimulatorAccess<dim>
     {
       public:
         /**
+         * Initialization function. This function is called once at the
+         * beginning of the program after parse_parameters is run and after
+         * the SimulatorAccess (if applicable) is initialized.
+         */
+        void initialize ();
+
+        /**
+         * Connect the topography function to the pre_set_initial_state signal.
+         */
+        void
+        connect_to_signal(SimulatorSignals<dim> &signals);
+
+        /**
+         * Relocate the vertical coordinate of the given point based on
+         * the topography at the surface specified by the initial topography
+         * model.
```

Comment:
Please describe the difference between the two functions below or separate the documentation into two blocks, one per function.